torch/csrc/jit/passes/onnx/remove_inplace_ops_for_onnx.cpp

```diff
@@ -700,9 +711,7 @@ void InplaceConverter::gatherAttrNameInitialValueMap(
 //    %1 : __torch__.___torch_mangle_1.M = prim::CreateObject()
 //    ...
 //    %10 : Tensor = aten::arange(%6, %7, %7, %7, %7)
-//    %26 : bool = prim::Constant[value=0]()
-//    %27 : Tensor?[] = prim::ListConstruct()
-//    %28 : Tensor = aten::index_put_(%19, %27, %10, %26)
+//    %28 : Tensor = aten::set_(%19, %10)
```

Comment:
This is cool! It uses set_ to simplify the graph.

Reply:
yes while trying to find a replacement for SetAttr on lists I found this op, which looks like a cleaner replacement for tensor attributes too.